app/contact/page.tsx

```
@@ -0,0 +1,140 @@
+import { Textarea } from "@/components/ui/textarea";
+import { useToast } from "@/components/ui/use-toast";
+
+export default function ContactPage() {
+  const [name, setName] = useState("");
+  const [email, setEmail] = useState("");
+  const [reason, setReason] = useState("");
+  const [message, setMessage] = useState("");
+  const { toast } = useToast();
+
+  const [isLoading, setIsLoading] = useState(false);
+  const { toast } = useToast();
```

Biome (2.1.2)

[error] Shouldn't redeclare 'toast'. Consider to delete it or rename it.

'toast' is defined here:


(lint/suspicious/noRedeclare)